src/invoices/InvoicesList/Invoice/utils.js

```diff
@@ -1,11 +1,23 @@
-import { get } from 'lodash';
+import { find, get } from 'lodash';
 
 import { INVOICE_STATUS } from '../../../common/constants';
 
 // eslint-disable-next-line import/prefer-default-export
-export const createInvoiceLineFromPOL = (poLine, invoiceId) => {
+export const createInvoiceLineFromPOL = (poLine, invoiceId, vendor) => {
   const quantityPhysical = get(poLine, 'cost.quantityPhysical', 0);
   const quantityElectronic = get(poLine, 'cost.quantityElectronic', 0);
+  const accountNumber = get(poLine, 'vendorDetail.vendorAccount');
+  const optionalProps = {};
+
+  if (accountNumber) {
+    const accounts = get(vendor, 'accounts', []);
+    const accountingCode = get(find(accounts, { accountNo: accountNumber }), 'appSystemNo', '') || get(vendor, 'erpCode');
```

Comment:
codestyle

Reply:
what do you mean?

Reply:
long line

Reply:
line is changed - comment doesn't make sense